apps/submission/io/pixel.py

```diff
@@ -0,0 +1,180 @@
+import pandas
+
+from django.utils.translation import ugettext as _
+
+from apps.core.models import OmicsUnit, Pixel, PixelSet
+from apps.data.models import Entry
+from ..exceptions import PixelSetParserError, PixelSetParserSaveError
+
+
+class PixelSetParser(object):
+
+    def __init__(self,
+                 pixelset_path,
+                 description=None,
+                 analysis=None,
+                 omics_unit_type=None,
+                 strain=None):
+
+        self.pixelset = None
+        self.pixelset_path = pixelset_path
+        self.description = description
+        self.analysis = analysis
+        self.omics_unit_type = omics_unit_type
+        self.strain = strain
+        self.pixels = None
+        self.db_pixels = {
+            'new': [],
+            'update': [],
+        }
+
+    def parse(self, force=False):
+
+        if self.pixels is not None and not force:
+            return
```

Comment:
Maybe we should be logging this, like `aborting parse() because not pixels and force = False`

Reply:
At which level: `DEBUG` or `INFO`?

Reply:
`DEBUG` I think